pages/common/pueue.md

```diff
@@ -3,34 +3,18 @@
 > Pueue is a command-line task management tool for sequential and parallel execution of long-running tasks.
 > More information: <https://github.com/Nukesor/pueue>.
 
-- Add a command to the task list:
+- Show general help and available subcommands:
 
-`pueue add -- "{{command}}"`
+`pueue --help`
 
-- List tasks in the task list:
+- Check pueue version:
 
-`pueue status`
+`pueue --version`
 
-- Send data to a task's stdin:
+- Execute a pueue subcommand:
 
-`pueue send {{task_id}} {{"hello"}}`
+`pueue {{subcommand}}`
 
-- View a task's stdout and stderr, as well as basic information about that task:
+- Show more example on a pueue subcommand:
```

Comment:
Let's remove this one.  We don't typically include `tldr` examples in multi-subcommand pages.